NEWS

```diff
@@ -1,3 +1,18 @@
+08-Jan-2021 Ignition 2.9.0
+
+  Changes:
+
+    - Require filesystems.format if wipeFilesystem or mountOptions is specified
+    - Refactor code to address golangci-lint warnings
+
+  Bug Fixes:
+
+    - Fix fetching configs from S3 resources when running on non-default AWS partitions
+    - Support fetching userdata from IMDSv2 on AWS
```

Comment:
Hmm.  If this was new functionality, it should go under Features.  But 2.4.0 claimed to support IMDSv2, so actually your original "Fix" phrasing was the right way to go.  "Fix userdata fetch from IMDSv2 on AWS"?

Reply:
ok